account_asset_management_xlsx/report/report_account_asset_xlsx.py

```diff
@@ -0,0 +1,459 @@
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.fr/>)
+# @author: La Louve
+# Copyright (C) 2020-Today: Druidoo (<https://www.druidoo.io>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from xlsxwriter.utility import xl_rowcol_to_cell
+
+from odoo import SUPERUSER_ID, fields, models
+from odoo.api import Environment
+
+from .image_util import get_record_image_path
+
+
+class ReportAccountAssetXlsx(models.AbstractModel):
+    _inherit = "report.report_xlsx.abstract"
+    _name = "report.report_account_asset_xlsx"
+    _description = "Report Account Asset XLSX"
+
+    def create_xlsx_report(self, ids, data):
+        self.env = Environment(self.env.cr, SUPERUSER_ID, self.env.context)
+        return super().create_xlsx_report(ids, data)
+
+    def generate_xlsx_report(self, workbook, data, objects):
+        wizard_rec = objects and objects[0] or objects
+        wizard_rec = self.env["account.asset.xlsx.wizard"].browse(wizard_rec.id)
+
+        resources = dict(**self._context)
+        resources.update(
+            {
+                "row_pos": 5,
+                "workbook": workbook,
+            }
+        )
+        resources.update(self._define_formats(workbook))
+
+        sheet = workbook.add_worksheet()
+        resources.update({"sheet": sheet})
+
+        self._set_cells_size(resources)
+        self.generate_report_title(resources)
+
+        resources = self.generate_report_general(wizard_rec, resources)
+
+        profile_datas_lst = wizard_rec and wizard_rec.get_profile_datas() or []
+        resources.update(
+            {
+                "summary_column_info": {
+                    "value": [],
+                    "value_residual": [],
+                    "salvage_value": [],
+                    "amo_ant": [],
+                    "amo_de_lan": [],
+                    "cum_amo": [],
+                    "val_nette": [],
+                }
+            }
+        )
+        for profile_datas in profile_datas_lst:
+            resources = self.generate_report_profile(profile_datas, resources)
+
+        resources = self.generate_report_summary(resources)
+
+    def _set_cells_size(self, resources):
+        sheet = resources["sheet"]
+        sheet.set_default_row(20)
+        sheet.set_column("A:Z", None, resources["fm_default"])
+        sheet.set_column("A:A", 40)
+        sheet.set_column("B:L", 20)
+
+    def generate_report_title(self, resources):
+        company_logo_path = self._get_company_logo_path()
+        sheet = resources["sheet"]
+        if company_logo_path:
+            sheet.insert_image(0, 0, company_logo_path)
+        sheet.merge_range(
+            "B2:K5",
+            "Immobilisations et amortissements",
```

Comment:
```suggestion
            "Assets And Depreciation",
```
translate `Assets And Depreciation` to `Immobilisations et amortissements` in i18n/fr.po

Note: the module must be in English and translated to French in i18n/fr.po